cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -607,6 +616,118 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
                         </Button>
                     </CVATTooltip>
                 </Col>
+                <Col>
+                    <CVATTooltip title='Contrast Limited Adaptive Histogram Equalization' className='cvat-opencv-image-tool'>
+                        <Button
+                            className={
+                                hasFilter(filters, ImageFilterAlias.CLAHE) ?
+                                    'cvat-opencv-clahe-tool-button cvat-opencv-image-tool-active' : 'cvat-opencv-clahe-tool-button'
+                            }
+                            onClick={(e: React.MouseEvent<HTMLElement>) => {
```

Comment:
The code for enabling the filter (and for the histogram as well) could be extracted into a separate function to reduce duplication